File: Algorism/src/Algorism/Python/programmers/greedy/P68644.py
def solution(numbers):
    answer = set()
    for i in range(0, len(numbers)):
        for j in range(1, len(numbers)):
            if i == j:
                continue
            else:
                answer.add(numbers[i]+numbers[j])
    return sorted(answer)


print(solution([2, 1, 3, 4, 1]))

File: Algorithm/src/Algorithm/Java/programmers/greedy/P68644.java
package Algorithm.Java.programmers.greedy;

import java.util.Arrays;
import java.util.HashSet;
import java.util.Set;

public class P68644 {
    public static void main(String[] args) {

        System.out.println(Arrays.toString(solution(2, 1, 3, 4, 1)));
    }

    public static int[] solution(int... numbers) {
        Set<Integer> set = new HashSet<>();
        for (int i = 0; i < numbers.length; i++) {
            for (int j = 0; j < numbers.length; j++) {
                if (i != j) {
                    set.add(numbers[i] + numbers[j]);
                }
            }
        }
        return set.stream().sorted().mapToInt(Integer::intValue).toArray();
    }
}
